### DISCUSSION TITLE
<< Installation/requirements >>

### DISCUSSION THREAD

**Discussion by icamps** [SakanaAI/AI-Scientist#72] [Q&A]
Hi.

I read the installation and the requirements instructions, but there are two things that are not clear for me, and I appreciate it if you can clarify them:

1. How many LLM's should be installed/used in order for AI-Scientist to work.
2. Minimum hardware requirements to run AI-Scientist.

Thanks in advance.
Camps

**Comment by BradKML** [comment]
Seconding this but also full local LLM support (see Ollama)
